integrations/docker/images/chip-build-vscode/Dockerfile

```diff
@@ -26,4 +29,6 @@ ENV EFR32_SDK_ROOT=/opt/SiliconLabs/sdk_support
 ENV EFR32_BOARD=BRD4161A
 ENV ANDROID_HOME=/opt/android/sdk
 ENV ANDROID_NDK_HOME=/opt/android/android-ndk-r21b
+ENV MBED_GCC_ARM_PATH=/opt/mbed-os-toolchain/gcc-arm-none-eabi-9-2019-q4-major/bin/
```

Comment:
Isn't this the same as /opt/ARM-software/gcc-arm-none-eabi-9-2019-q4-major ?

Reply:
Yes, you're right. 
The question is do we want to have separated ARM toolchains for nrf and mbed-os or we want to stay with one version, shared between these two platforms.
It's possible that either nrf or mbed-os would like to switch to another toolchain version in the future, 

If we wanted to stay with with one toolchain than we would have to move gcc-arm-xxxx installation to chip-build's Dockerfile, instead of installing it in chip-build-mbed-os and chip-build-nrf-platform images. 
It will definitely reduce chip-build-vscode image size which would be nice.

From other hand I saw that in scripts/pigweed.json there is listed another gcc-arm toolchain version for installation...

Reply:
Changing the version would require a change the docker file anyway right?

We could have
   /opt/ARM-software/gcc-arm-none-eabi-9-2019-q4-major
   /opt/ARM-software/gcc-arm-none-eabi-whatever

They should go in the same structure, since they would be different versions of the same product.